deploy/airgap-prepare.sh

```diff
@@ -55,7 +55,7 @@ for image in $(grep "image: docker.io" deployment.yaml | awk -F' ' '{ print $2 }
   docker pull $image
 done
 # Save all referenced images into a tarball.
-grep "image: " deployment.yaml | awk -F' ' '{ print $2 }' | xargs docker save -o $CRED_SHIELD_IMAGES_TAR
+grep "image: docker.io" deployment.yaml | awk -F' ' '{ print $2 }' | xargs docker save -o $CRED_SHIELD_IMAGES_TAR
```

Comment:
this grep should be saving all images, including internal ones for proxy/tenant services

Reply:
fixed